<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="10,1">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="MainGrid" stretch="0">
      <item>
       <widget class="QTabWidget" name="TabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="BooksTab">
         <attribute name="title">
          <string>Books</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,3">
          <item>
           <widget class="QTableWidget" name="BooksDisplayGrid"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
            <item>
             <widget class="QLabel" name="AddBookLabel">
              <property name="text">
               <string>Add a book</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="5" column="0">
               <widget class="QLabel" name="AuthorLabel">
                <property name="text">
                 <string>Author</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="TitleLabel">
                <property name="text">
                 <string>Title</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="PublisherLabel">
                <property name="text">
                 <string>Publisher</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="ISBNLabel">
                <property name="text">
                 <string>ISBN</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QComboBox" name="AuthorComboBox"/>
              </item>
              <item row="1" column="1">
               <widget class="QPlainTextEdit" name="SynopsisTextEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="YearLabel">
                <property name="text">
                 <string>Year of release</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="SynopsisLabel">
                <property name="text">
                 <string>Synopsis</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="TitleLineEdit">
                <property name="maxLength">
                 <number>128</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="RatingLabel">
                <property name="text">
                 <string>Rating 1-5</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="YearSpinBox">
                <property name="maximum">
                 <number>2050</number>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="RatingSpinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="ISBNLineEdit">
                <property name="maxLength">
                 <number>32</number>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QComboBox" name="PublisherComboBox"/>
              </item>
              <item row="7" column="0">
               <widget class="QPushButton" name="AddBookButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="AuthorsTab">
         <attribute name="title">
          <string>Authors</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="PublishersTab">
         <attribute name="title">
          <string>Publishers</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="InfoGrid" columnstretch="2,1,1,5">
      <item row="0" column="2">
       <widget class="QLabel" name="StatusLabel">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="LoginButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="Status">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="LastAction">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="UsernameFrom">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>64</number>
        </property>
        <property name="placeholderText">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="PasswordForm">
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>64</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="LastActionLabel">
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="RegisterButton">
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
